queue/abstract/driver/fifottl.lua

```diff
@@ -300,12 +300,18 @@ function method.release(self, id, opts)
     if task == nil then
         return
     end
+    if opts.touch_ttl == nil then opts.touch_ttl = true end
```

Comment:
We usually cover all new functionality with a test. Aren't you mind to add one? They're in the `t/` directory.

Reply:
Covered in new commit. Sorry for that